### PR TITLE
Enable softmax tests that pass locally

### PR DESCRIPTION
**Pull request by ashishfarmer**

### CODE REVIEW

**Comment by ashishfarmer** [comment]
FP16 softmax failure with error 5.62072e-05

@pytorchbot retest this please
